CONTRIBUTING.md

```diff
@@ -49,16 +49,20 @@ The Git LFS teams mark issues and pull requests with the following labels:
 * `roadmap` - A checklist issue with tasks to fulfill something from the
 [roadmap](./ROADMAP.md)
 
+## Branching strategy
+
+In general, contributors should develop on branches based off of `master` and pull requests should be to `master`.
+
```

Comment:
I think we should just add that separately, rather than dictating how it all works to @dakotahawkins so he can just type what we said :)